docs/src/developer_guide.md

```diff
@@ -2,7 +2,7 @@
 
 This page provides additional documentation on abstractions to use
 `ClimaDiagnostics`. Before reading this page, make sure you are familiar with
-the terminology. You know to know what a [`DiagnosticVariable`](@ref ClimaDiagnostics.DiagnosticVariables.DiagnosticVariable) and a
+the terminology. You need to know what a [`DiagnosticVariable`](@ref ClimaDiagnostics.DiagnosticVariables.DiagnosticVariable) and a
 [`ScheduledDiagnostic`](@ref ClimaDiagnostics.ScheduledDiagnostics.ScheduledDiagnostic) are.
 
 There are three components needed to add support for `ClimaDiagnostics.jl` in your package.
```

Comment:
@Sbozzolo Here, we say "three components", but we then only list two. (or maybe I am missing something?) - I think maybe 1. should be DiagnosticVariable?